addon/core/keymap.ts

```diff
@@ -29,7 +29,46 @@ import { liftEmptyBlockChecked } from '@lblod/ember-rdfa-editor/commands/lift-em
 
 export type Keymap = (schema: Schema) => Record<string, Command>;
 
+const reduceParagraphIndent: Command = (state, dispatch) => {
+  if (!(state.selection instanceof TextSelection)) {
+    return false;
+  }
+
+  const { $cursor } = state.selection;
+
+  if (
+    !$cursor ||
+    // Skip action at the start of document
+    $cursor.pos === 0 ||
+    // Skip action if cursor is not at the first position of "child"
+    $cursor.parentOffset !== 0 ||
+    // Skip action node has no existing "indentationLevel"
+    !$cursor.parent.attrs.indentationLevel
+  ) {
+    return false;
+  }
+
+  const paragraphPosition = $cursor.before($cursor.depth);
+  const currentIndentationLevel = $cursor.parent.attrs
+    .indentationLevel as number;
+
+  if (dispatch) {
+    const tr = state.tr.setNodeAttribute(
+      paragraphPosition,
+      'indentationLevel',
+      currentIndentationLevel - 1
+    );
+
+    dispatch(tr);
+
+    return true;
+  }
+
+  return false;
```

Comment:
In this case you should also return `true`. When `dispatch` is not defined running the command only checks whether the operation is possible, in this case it is.